autosubmit/autosubmit.py

```diff
@@ -737,7 +735,7 @@ def run_command(args):
                                       args.txt_logfiles, args.profile, detail=False)
         elif args.command == 'stats':
             return Autosubmit.statistics(args.expid, args.filter_type, args.filter_period, args.output,
-                                         args.section_summary, args.jobs_summary, args.hide, args.notransitive, args.database)
```

Comment:
This was the first bug, the one reported in the issue linked here. The `args.database` was removed from the function signature, but not from the call or args.